functions/webhooks/taskrouterCallback.protected.ts

```diff
@@ -78,17 +66,42 @@ const isCleanupCustomChannel = (eventType: EventType, taskAttributes: { channelT
   return channelToFlex.isAseloCustomChannel(taskAttributes.channelType);
 };
 
+/**
+ * Fetch all taskrouter listeners from the listeners folder
+ */
+const getListeners = () => {
+  const functionsMap = Runtime.getFunctions();
```

Comment:
We could definitely make this logic a little cleaner if we merge the shared module stuff, we could have a `register` method here and all the `TaskRouterListeners` register themselves with the `taskRouterCallback`. But what you have here is definitely the best approach within the current limitations